extensions/amp-accordion/amp-accordion.md

```diff
@@ -120,7 +120,7 @@ The events below will be triggered on `section`s of `accordion`.
   </tr>
   <tr>
     <td width="40%"><strong><code>expanded</code></strong></td>
-    <td>Set this attribute on a <code>&lt;section&gt;</code> to display the section as expanded on page load.</td>
+    <td>Set this attribute or bind to <code>[expanded]</code> on a <code>&lt;section&gt;</code> to display the section as expanded on page load.</td>
```

Comment:
We should remove the part about binding here, for two reasons:

1. The attribute name is different now
2. Bindings do not run on page load, the wording here makes it sound like they do